components/CatalogNavigation.vue

```diff
@@ -27,7 +27,7 @@
 </script>
 
 <template>
-    <div class="navigation">
+    <div class="navigation p5">
```

Comment:
It is, otherwise the font-size is different.
Using the custom-classes like p5 automatically adjusts changed font-sizes between desktop and mobile.
With changes in widths the one smaller font-size is necessary to use.